content/v2.2/database/overview.md

```diff
@@ -137,9 +137,8 @@ module Main
 end
 ```
 
-But you don’t need to define every struct ahead of time, only to extend its functionality. Structs will be generated on-demand in your slice’s `Structs` namespace.
+But you don’t need to define every struct ahead of time, only to extend its functionality. If you don't define a Struct class, Structs will be generated on-demand in the appropriate namespace.
 
-A Struct is not a permanent abstraction of a piece of data: it is a momentary projection of the data you requested. This means that instead of a User model that fills every role you need of a user, you could project user data as a Credential for authentication, a Role for authorization, a Visitor for displaying their identity on the page. Every projection can serve a specific purpose, and contain exactly the information you need and nothing more.
+A Struct is not a permanent abstraction of a piece of data: it is a momentary projection of the data you requested. This means that instead of a `User` model that fills every role you need of a user, you could project user data as a `Credential` for authentication, a `Role` for authorization, a `Visitor` for displaying their identity on the page. Every projection can serve a specific purpose, and contain exactly the information you need and nothing more. The way to map them to different object is [using `#map_to` on the Relation](https://rom-rb.org/learn/core/5.2/structs/#mapping-to-custom-objects).
```

Comment:
```suggestion
A Struct is not a permanent abstraction of a piece of data: it is a momentary projection of the data you requested. This means that instead of a `User` model that fills every role you need of a user, you could project user data as a `Credential` for authentication, a `Role` for authorization, a `Visitor` for displaying their identity on the page. Every projection can serve a specific purpose, and contain exactly the information you need and nothing more.
```

"object" should be plural here, but I think `map_to` belongs in the forthcoming Repository guide, this seems a little advanced for the Overview.